src/orquestra/sdk/_base/_env.py

```diff
@@ -40,6 +40,11 @@
 Set by Studio to share current project ID
 """
 
+CURRENT_USER_ENV = "ORQ_CURRENT_USER"
+"""
+Set by Studio to share current project ID
```

Comment:
**nitpick**: looks like a copy+paste error. While at it, could you also edit the docstrings of the other env variables? Nowadays it's set by the Driver as well (I think?)

```suggestion
Set by Studio and CE.
```